Detectors/ITSMFT/MFT/reconstruction/src/TrackerTask.cxx

```diff
@@ -0,0 +1,99 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file  TrackerTask.cxx
+/// \brief Task driving the track finding from MFT clusters
+/// \author bogdan.vulpescu@cern.ch
+/// \date Feb. 7, 2018
+
+#include "MFTReconstruction/TrackerTask.h"
+#include "DataFormatsITSMFT/Cluster.h"
+#include "SimulationDataFormat/MCCompLabel.h"
+#include "SimulationDataFormat/MCTruthContainer.h"
+
+ClassImp(o2::MFT::TrackerTask)
+
+  using namespace o2::MFT;
```

Comment:
delete blanks at beginning of line, strange that this went through the format checker

Reply:
Actually the blank is introduced by clang-format. I trusted him.

Reply:
ok, then it's a configuration problem in clang-format

Reply:
The issue is that the ClassImp has no ';' at the end; clang-format gets confused by this;

I usually try to put either the ';' or move ClassImp to the end. Unfortunately, this shows that one can not trust clang-format out of the box ...